locustfile.py

```diff
@@ -0,0 +1,239 @@
+from locust import HttpUser, task, between
+import os
+import logging
+from utils.env_helper import load_env_variables
```

Comment:
Reorder imports.

It should always be python native libraries i.e. os and logging, followed by third party libraries such as locust, and then code from us (i.e. utils.env_helper).